<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advance club - Funnel</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="questionText">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="answerText">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonAnswer">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Answer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonNotSure">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>I'm not sure</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonYes">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>I'm right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonNo">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>I'm wrong</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>545</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_test"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_test">
   <property name="text">
    <string>Open test</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
